src/main/java/kitchenpos/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package kitchenpos.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String message) {
```

Comment:
```suggestion
    public EntityNotFoundException(ErrorCode code) {
```

다양한 방법이 있겠지만, 커스텀 exception을 만든 만큼 특별한 책임을 좀더 부여하면 어떨까요?

[커스텀 exception](https://stackify.com/java-custom-exceptions/)
참고해보시면 좋을것 같아요!

Reply:
넵 다음 단계 진행하면서 작업해보도록 하겠습니다! ㅎㅎ 좋은 방법 알려주셔서 감사합니다 ☺️